src/main/resources/net/uaznia/lukanus/hudson/plugins/gitparameter/javascript/git-parameter-select.js

```diff
@@ -4,74 +4,99 @@ function gitParameterUpdateSelect(listBox, url, divId, config) {
 
-        if (originalOnSuccess !== undefined)
-            originalOnSuccess(rsp);
+            if (originalOnSuccess !== undefined)
+                originalOnSuccess(rsp);
 
-        var errors = eval('(' + rsp.responseText + ')').errors
-        let error_div = $("git_parameter_errors_" + divId);
-        if (errors.length !== 0) {
-            error_div.show();
-            error_div.addClassName("error");
-            let $error_ul = $("git_parameter_errors_ul_" + divId);
-            var lis = "";
-            for (var j = 0; j < errors.length; j++) {
-                lis += "<li>" + escapeHTML(errors[j]) + "</li>"
+            var errors = json.errors
+            let error_div = document.getElementById("git_parameter_errors_" + divId);
+            if (errors.length !== 0) {
+                error_div.style.display = "";
+                error_div.classList.add("error");
+                let $error_ul = document.getElementById("git_parameter_errors_ul_" + divId);
+                var lis = "";
+                for (var j = 0; j < errors.length; j++) {
+                    lis += "<li>" + escapeHTML(errors[j]) + "</li>"
+                }
+                $error_ul.update(lis);
+            } else {
+                error_div.style.display = "none";
             }
-            $error_ul.update(lis);
-        }
-        else {
-            error_div.hide()
-        }
-
+        });
     };
     config.onFailure = function (rsp) {
-        l.removeClassName("select-ajax-pending");
-        status.innerHTML = rsp.responseText;
-        if (status.firstChild) {
-            status.firstChild.setAttribute('data-select-ajax-error', 'true')
-        }
-        Behaviour.applySubtree(status);
-        // deleting values can result in the data loss, so let's not do that unless instructed
-        var header = rsp.getResponseHeader('X-Jenkins-Select-Error');
-        if (header && "clear" === header.toLowerCase()) {
-            // clear the contents
-            while (l.length > 0) l.options[0] = null;
-        }
+        rsp.text().then(text => {
+            l.classList.remove("select-ajax-pending");
+            status.innerHTML = text;
+            if (status.firstChild) {
+                status.firstChild.setAttribute('data-select-ajax-error', 'true')
+            }
+            Behaviour.applySubtree(status);
+            // deleting values can result in the data loss, so let's not do that unless instructed
+            var header = rsp.headers.get('X-Jenkins-Select-Error');
+            if (header && "clear" === header.toLowerCase()) {
+                // clear the contents
+                while (l.length > 0) l.options[0] = null;
+            }
+        });
 
     };
 
-    l.addClassName("select-ajax-pending");
-    new Ajax.Request(url, config);
+    l.classList.add("select-ajax-pending");
+
+    fetch(url, {
+        method: 'POST',
+        body: stringify(config.parameters),
+        headers: crumb.wrap({})
```

Comment:
The original Prototype version was sending this as `Content-type: application/x-www-form-urlencoded`, while the version in this PR is using a different content type. To avoid the risk of regression, I think it is safer to keep the behavior the same as before. See https://github.com/jenkinsci/jenkins/pull/7982 for an example of how to do this.